src/OrchardCore.Build/Dependencies.props

```diff
@@ -10,6 +13,7 @@
     <FluidCoreVersion>1.0.0-beta-9399</FluidCoreVersion>
     <YesSqlVersion>2.0.0-beta-1097</YesSqlVersion>
     <YesSqlProvidersVersion>1.0.0-beta-1097</YesSqlProvidersVersion>
+    <NewtonsoftJson>10.0.3</NewtonsoftJson>
```

Comment:
Maybe `NewtonsoftJson` => `NewtonsoftJsonPackageVersion`.

Reply:
Not sure, `PackageVersion` is just here because this is how the ASP.NET team generated the file. We don't have to follow it for the other dependencies. But we can surely add the `Version` suffix to be consistent.